DataManagementSystem/Utilities/CatalogUtilities.py

```diff
@@ -0,0 +1,138 @@
+########################################################################
+# $HeadURL$
+########################################################################
+
+""" DIRAC FileCatalog utilities
```

Comment:
So why not put that one in /Resources/Catalog/Utilities? I understand it is different from Storage, no problem with that...